src-cpp/HandleImpl.cpp

```diff
@@ -123,6 +122,34 @@ int RdKafka::open_cb_trampoline (const char *pathname, int flags, mode_t mode,
   return handle->open_cb_->open_cb(pathname, flags, static_cast<int>(mode));
 }
 
+int RdKafka::cert_verify_cb_trampoline(unsigned char* cert, long len, void *opaque)
+{
+    RdKafka::HandleImpl *handle = static_cast<RdKafka::HandleImpl *>(opaque);
+
+#if WITH_SSL
+    return handle->cert_verify_cb_->cert_verify_cb(cert, len);
+#else
+    RD_UNUSED(cert);
```

Comment:
I can certainly remove these.  I was trying to be consistent with other places in the code which ifdef WITH_SSL such the config which sets these in rdkafka_conf.h/c.  On a side note I cannot seem to compile code these days without WITH_SSL being defined so this may not be necessary anymore.

Reply:
I mean the RD_UNUSED, I haven't seen any compilers warn about that.

As for !WITH_SSL: I'll investigate, thanks for letting me know!

Reply:
I will remove for now.  If the compiler barks about this I may need to add it back.